app/controllers/import_export_dates_controller.rb

```diff
@@ -1,7 +1,9 @@
 class ImportExportDatesController < ApplicationController
   include DeclarableHelper
 
-  before_action :disable_search_form
+  before_action :disable_search_form, :disable_switch_service_banner do
```

Comment:
:smile: This is a new trick I didn't know about - symbols and block, nice! definitely one to remember